packages/atlas/src/components/Section/SectionFooter/SectionFooter.tsx

```diff
@@ -43,7 +45,11 @@ export const SectionFooter = (props: SectionFooterProps) => {
   const { ref, inView } = useInView()
 
   useEffect(() => {
-    if ((props.type === 'infinite' || (props.type === 'load' && isSwitchedToInfinite)) && inView && !isLoading) {
+    if (
+      ((props.type === 'infinite' && !props.reachedEnd) || (props.type === 'load' && isSwitchedToInfinite)) &&
```

Comment:
shouldn't we move `props.reachedEnd` to the end of this condition? I could be wrong, but I think if the footer has type `load`, we should use `props.reachedEnd` as well

Reply:
I covered this in Channel section PR, so no reason to duplicate code